src/Internal/Helpers.purs

```diff
@@ -282,3 +290,23 @@ concatPaths a b =
   right = fromMaybe b (stripPrefix (Pattern "/") b)
 
 infixr 5 concatPaths as <</>> -- </> is taken
+
+-- | Runs two `Aff` actions concurrently
+-- | Get resolved by the first which either resolves or throws
+-- | Expected properties:
+-- | race x y = race y x
+-- | race x (race y z) = race (race x y) z
+-- | race never x = x
+race :: forall (a :: Type). Aff a -> Aff a -> Aff a
+race f g = liftEither =<< sequential (parallel (try f) <|> parallel (try g))
+
+-- | Runs multiple `Aff` actions concurrently
+-- | raceMany [] = never
+-- | raceMany [f] = f
+-- | raceMany [f, g, h] = race (race f g) h
+raceMany
+  :: forall (a :: Type) (f :: Type -> Type). Foldable f => f (Aff a) -> Aff a
+raceMany = foldl race never
+
+delaySec :: Seconds -> Aff Unit
+delaySec (Seconds seconds) = delay $ Milliseconds (seconds * 1000.0)
```

Comment:
There's also

```suggestion
delaySec :: Seconds -> Aff Unit
delaySec seconds = delay $ convertDuration seconds
```

which helps with avoiding these conversion constants

Reply:
That's a good one!